101-azure-api-management-create/azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters":
+    {
+        "publisherEmail":
```

Comment:
Every parameter needs a metadata.description tag. Please review the contribution guide https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md